corehq/util/quickcache.py

```diff
@@ -290,7 +290,7 @@ def domains_for_user(request):
             CacheWithTimeout(
                 django_caches['locmem'],
                 timeout=memoize_timeout)]
-        if timeout != 0:
+        if timeout > 0:
```

Comment:
no, not as the code currently stands.  Line 285 above:
```python
timeout = timeout if timeout is not None else 5 * 60
```
This means the only way to never timeout is to use something like `timeout=BIGNUM`, which is probably what we want.